Src/Newtonsoft.Json-for-Unity.Tests/Assets/Aot.Tests/AotTests.cs

```diff
@@ -5,46 +5,44 @@
 
 namespace Aot.Tests
 {
+    /// <summary>
+    /// Tests are only relevant for AOT builds, such as when using IL2CPP
+    /// </summary>
     [TestFixture]
     public class AotTests
     {
-#if ENABLE_IL2CPP
-        static AotTests()
-        {
-            var myAotEnsuredList = new List<MyAotEnsuredClass>();
-        }
-
-        class MyNonAotClass
+        private class MyNonAotClass
         {
-#pragma warning disable 649
+#pragma warning disable 649 // Field is never assigned to, and will always have its default value `null'
             public string a;
         }
 
-        class MyAotEnsuredClass
+        private class MyAotEnsuredClass
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private type 'MyAotEnsuredClass'.](https://app.codacy.com/app/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4554595)